house/forms.py

```diff
@@ -38,3 +39,13 @@ class ExpenseForm(forms.ModelForm):
     class Meta:
         model = Expenses
         fields = ('date', 'amount', 'category', 'description')
+
+
+class YearFilterForm(forms.Form):
```

Comment:
Did you implement the logic of this form

Reply:
Yes, this form filter the years from 2020 to the current year
I have changed it a bit, maybe now it would be more understandable
What do you think? @ronyyosef

Reply:
I think this method could be in inline but thats just my own preference

Reply:
I think it's more understandable now 😇

Reply:
Looks great!